src/test/java/org/folio/circulation/resources/RequestQueueResourceTest.java

```diff
@@ -0,0 +1,315 @@
+package org.folio.circulation.resources;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.folio.circulation.support.http.client.Response;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import api.support.APITests;
+import api.support.builders.ReorderQueueBuilder;
+import api.support.builders.RequestBuilder;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+@RunWith(JUnitParamsRunner.class)
+public class RequestQueueResourceTest extends APITests {
+
+  private IndividualResource item;
+  private IndividualResource jessica;
+  private IndividualResource steve;
+  private IndividualResource james;
+  private IndividualResource rebecca;
+  private IndividualResource charlotte;
+
+  @Before
+  public void setUp() throws Exception {
+    item = itemsFixture.basedUponSmallAngryPlanet();
+
+    jessica = usersFixture.jessica();
+    steve = usersFixture.steve();
+    james = usersFixture.james();
+    rebecca = usersFixture.rebecca();
+    charlotte = usersFixture.charlotte();
+  }
+
+  @Test
+  public void validationErrorOnItemDoNotExists() throws Exception {
+    Response response = requestQueueFixture.attemptReorderQueue(
+      UUID.randomUUID().toString(),
+      new ReorderQueueBuilder()
+        .addReorderRequest(UUID.randomUUID().toString(), 1)
```

Comment:
Why does there need to be a request in this example? Is it not sufficient for only the item ID to provided to trigger this error?

Reply:
Yep, it is not mandatory for this test, but just to be sure that the cause is undefined Item ID, but not an empty reordered queue.